### DISCUSSION TITLE
Scroll group into view?

### DISCUSSION THREAD

**Discussion by maccman** [petyosi/react-virtuoso#667] [Q&A] [answered]
I'm using Virtuoso with groups. Is there a way to scroll a particular group into view?

**Comment by petyosi** [comment]
Hey,

Not exactly. In the grouped mode, groups are more like "floating" markers. However, if you scroll the first item of the group into view, the group should be correctly positioned. 

Let me know this makes sense.

**Reply by maccman** [reply]
That makes sense. The issue that often the group is often blank. See below - I'm trying to scroll to the current day's tasks.

![CleanShot 2022-05-21 at 10 12 42@2x](https://user-images.githubusercontent.com/2142/169655498-de822fb5-246d-4c0f-a2bd-de65108cee34.png)

**Reply by petyosi** [reply]
I see. Let me think a bit about it, should be fairly easy to expose an API.

**Reply by maccman** [reply]
Thanks - you are epic! 🙌

**Comment by petyosi** [comment, marked as answer]
So, with 2.12.0 you can now pass either `groupIndex: number` or `index: number` as a field in the location argument for both `scrollToIndex` and `scrollIntoView`. 

Check this: https://github.com/petyosi/react-virtuoso/commit/92f61458fd85c2471ed0e623fbfbcd742ed793fd#diff-c21e4a507f43c5382f1f2741ac2e16818829326f8e90d26bf371433c5379f2c3

**Reply by maccman** [reply]
Incredible. I can't think of an open-source project with better support than Virtuoso.
